cylc/flow/config_schema.py

```diff
@@ -0,0 +1,609 @@
+            'login hosts': [VDR.V_STRING, ''],
+            'batch system': [VDR.V_STRING, ''],
+            'cylc executable': [VDR.V_STRING, ''],
+            'global init-script': [VDR.V_STRING, ''],
+            'copyable environment variables': [VDR.V_STRING, ''],
+            'retrieve job logs': [VDR.V_STRING, ''],
+            'retrieve job logs command': [VDR.V_STRING, ''],
+            'retrieve job logs max size': [VDR.V_STRING, ''],
+            'retrieve job logs retry delays': [VDR.V_STRING, ''],
+            'task event handler retry delays': [VDR.V_STRING, ''],
+            'tail command template': [VDR.V_STRING, ''],
+            'owner': [VDR.V_STRING_LIST],
+            'batch systems': {
+                'err tailer': [VDR.V_STRING, ''],
+                'out tailer': [VDR.V_STRING, ''],
+                'err viewer': [VDR.V_STRING, ''],
+                'out viewer': [VDR.V_STRING, ''],
+                'job name length maximum': [VDR.V_STRING, ''],
+                'execution time limit': [VDR.V_INTERVAL_LIST],
+                'execution polling intervals': [VDR.V_INTERVAL_LIST],
+                'execution retry delays': [VDR.V_INTERVAL_LIST],
+                'batch submit command template': [VDR.V_STRING]
+            },
+            'default directives': {
+                '__MANY__': [VDR.V_STRING],
+            },
+        },
+    },
+    'email': {
+        'mail events': [VDR.V_STRING_LIST, None],
+        'mail from': [VDR.V_STRING],
+        'mail smtp': [VDR.V_STRING],
+        'mail to': [VDR.V_STRING],
+        'mail footer': [VDR.V_STRING],
+        'monitor': {
+            'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
+        },
+    },
+    'scheduling': {
+        'initial cycle point': [VDR.V_CYCLE_POINT],
+        'final cycle point': [VDR.V_STRING],
+        'initial cycle point constraints': [VDR.V_STRING_LIST],
+        'final cycle point constraints': [VDR.V_STRING_LIST],
+        'hold after point': [VDR.V_CYCLE_POINT],
+        'cycling mode': (
+            [VDR.V_STRING, Calendar.MODE_GREGORIAN] +
+            list(Calendar.MODES) + ["integer"]),
+        'runahead limit': [VDR.V_STRING],
+        'max active cycle points': [VDR.V_INTEGER, 3],
+        'spawn to max active cycle points': [VDR.V_BOOLEAN],
+        'queues': {
+            'default': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+            '__MANY__': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+        },
+        'special tasks': {
+            'clock-trigger': [VDR.V_STRING_LIST],
+            'external-trigger': [VDR.V_STRING_LIST],
+            'clock-expire': [VDR.V_STRING_LIST],
+            'sequential': [VDR.V_STRING_LIST],
+            'exclude at start-up': [VDR.V_STRING_LIST],
+            'include at start-up': [VDR.V_STRING_LIST],
```

Comment:
Does anyone ever use these two?  They are (I think?) an ancient bodge made obsolete by `R1` graphs.

Reply:
(i.e. can we drop them?)

Reply:
Seems reasonable, but I'd like thoughts from @oliver-sanders and @dpmatthews

Reply:
These aren't deprecated so it would be pretty cheeky to drop them, however, none of the 600 suites currently running use these either...

Reply:
(again, by drop I meant deprecate :grimacing: ... we never just literally remove items)

Reply:
#3404